docs/process/ci_build.md

```diff
@@ -28,15 +28,24 @@ Basic concepts:
+There might be reasons why you want or need to test changes to GH Actions on your personal fork. Less frequently, there might be specific changes that cannot be tested via a PR because they only happen on merges to the main branch `master`. For example, API docs changes and benchmark dashboard changes should only occur on merges to master. In this case, you can use your personal fork of the upstream repo as a testing ground. The naive approach is not recommended -- to directly modify your `master` branch -- because it requires awkwardly changing your git repo during and after testing in ways that are often confusing and opposite to "git-flow" habits. 
+
+So here are steps to test GH Actions change on your personal fork, with links to an example:
+
+1. create a new testing-only branch that tacks on [an extra commit to make testing-appropriate changes](https://github.com/echeran/icu4x/pull/22/commits/538176500f54594cffa1844c0244c7135ea24b84)
```

Comment:
Suggestion: I generally prefer complete sentences with documentation, with proper capitalization and punctuation as it's easier to read an understand.

Reply:
Done. Thanks for the suggestion. I hadn't thought about that, but it makes sense, I'll keep that in mind for the future.